### DISCUSSION TITLE
Update/Edit Page shows nothing when adding Map fields into collection

### DISCUSSION THREAD

**Discussion by suhaimim** [directus/directus#9940] [Archive]
Countries Table List
![image](https://user-images.githubusercontent.com/35832684/142721593-e2304b41-e8e5-4486-b807-357da2fda366.png)

Selected Country Edit/Modify/Update Page
![image](https://user-images.githubusercontent.com/35832684/142721609-66363075-d113-4f43-a85c-2958e4b4b46a.png)

Adding Map Field into Countries Collection 
![image](https://user-images.githubusercontent.com/35832684/142721676-09b2e5a4-541b-4bae-afb5-5bd34c5f5ebf.png)

**Comment by rijkvanzanten** [comment]
Sorry I don't know what you mean.. if there's a bug in the app, please open a new issue detailing exactly how to reproduce it, and what errors you're experiencing 👍🏻

**Reply by suhaimim** [reply]
1. I create the COUNTRIES collection without Map Field
2. All working fine, and able to view/add/modify the COUNTRIES items (1st figure above)
3. Modify the COUNTRIES collection by adding the Map Field (3rd figure above)
4. Go back to the COUNTRIES content to modify the item in the list, but unable to view, return page not found. (2nd figure above)
5. When I remove back the Map field from COUNTRIES collection, all back to normal #2. 

The question, how to add the Map field into the collection without these error?
The title in this thread is more on the issue happed (aplogise for the confusing statement)

**Reply by suhaimim** [reply]
And I am not sure whether this is bug or my steps on the fields creations was wrong. 
Please advise.

**Reply by rijkvanzanten** [reply]
I'm not seeing that issue on my end. Would you be able to share a database dump so we can reproduce the issue locally?

**Reply by suhaimim** [reply]
The database is inside the docker image that i pull from directus. So, I dont know how to export/import or modify the db.

**Reply by rijkvanzanten** [reply]
See https://github.com/directus/directus/discussions/9970#discussioncomment-1682826

**Reply by suhaimim** [reply]
Replied on the same #9970 thread. Thank you.
